python/private/versions.bzl

```diff
@@ -15,32 +15,33 @@
 """The Python versions we use for the toolchains.
 """
 
-RELEASE_URL = "https://github.com/indygreg/python-build-standalone/releases/download/20211017"
-RELEASE_DATE = "20211017T1616"
+RELEASE_URL = "https://github.com/indygreg/python-build-standalone/releases/download/20220227"
+RELEASE_DATE = "20220227"
 
-# The integrity hashes can be computed with:
-# shasum -b -a 384 [downloaded file] | awk '{ print $1 }' | xxd -r -p | base64
```

Comment:
@UebelAndre I think we should keep this commentary, but agree it should be `256` algorithm and not `384`. @f0rmiga why 384?

Also @f0rmiga I think we should:

* have in the commentary an explanation of the fact that SRI format is required, as that's different to other places in Bazel.
* Use --long-options in the `shasum` example command not short ones. I checked and thankfully shasum support it.
* Explain why `xxd -r -p` is needed.

Reply:
I have no attachment as to whether or not this stays or is removed. I think the repository rule should report what value this should be should one not be provided though (which is something I fixed in this PR). Since this PR targets https://github.com/bazelbuild/rules_python/pull/618 I figured it'd be up to @f0rmiga as to what gets accepted and what doesn't.

Reply:
I'm okay with dropping the comment. Anyone dealing with Bazel at this level should know how to grab the hashes (there are many ways to accomplish this).

I'm also not particularly attached to which algorithm to use - this is an integrity check, not a user password in a database.

Reply:
@f0rmiga I think we _should keep the comment_. As a Bazel maintainer I can figure out how to generate the integrity strings, but it's just easier if the code documents this as it's not the trivial `shasum -a 256 foo` uses in other places in Bazel. 

I liked the commentary, and wanted it expanded address the bullet points above.

Reply:
https://github.com/bazelbuild/rules_python/pull/618/commits/d7ac042f2ad92670f60e25d4f97fcdef567c7859 addresses this more elegantly.

Reply:
Yep, looks good.